packages/prosess-vedtak/src/components/VedtakForm.jsx

```diff
@@ -1,437 +1,368 @@
+            }
+          : {},
+        skalBrukeOverstyrendeFritekstBrev: values?.[fieldnames.SKAL_BRUKE_OVERSTYRENDE_FRITEKST_BREV],
+        skalUndertrykkeBrev: values?.[fieldnames.SKAL_HINDRE_UTSENDING_AV_BREV],
+        isVedtakSubmission,
+        begrunnelserMedInformasjonsbehov: begrunnelser,
+        tilgjengeligeVedtaksbrev,
+      }));
+  };
 
-          {fritekstdokumenter?.length > 0 && <UstrukturerteDokumenter fritekstdokumenter={fritekstdokumenter} />}
+  const payload = values =>
+    aksjonspunkter
+      .filter(ap => ap.kanLoses)
+      .map(aksjonspunkt => ({
+        kode: aksjonspunkt.definisjon.kode,
+        begrunnelse: values?.[fieldnames.BEGRUNNELSE],
+        overstyrtMottaker: safeJSONParse(values?.[fieldnames.OVERSTYRT_MOTTAKER]),
+        fritekstbrev: {
+          brødtekst: values?.[fieldnames.BRØDTEKST],
+          overskrift: values?.[fieldnames.OVERSKRIFT],
+        },
+        skalBrukeOverstyrendeFritekstBrev: values?.[fieldnames.SKAL_BRUKE_OVERSTYRENDE_FRITEKST_BREV],
+        skalUndertrykkeBrev: values?.[fieldnames.SKAL_HINDRE_UTSENDING_AV_BREV],
+        isVedtakSubmission,
+        tilgjengeligeVedtaksbrev,
+      }));
 
-          {(isInnvilget(behandlingresultat.type.kode) || isDelvisInnvilget(behandlingresultat.type.kode)) && (
-            <VedtakInnvilgetPanel
-              intl={intl}
-              antallBarn={antallBarn}
-              behandlingsresultat={behandlingresultat}
-              readOnly={readOnly}
-              skalBrukeOverstyrendeFritekstBrev={skalBrukeOverstyrendeFritekstBrev}
-              ytelseTypeKode={ytelseTypeKode}
-              aksjonspunkter={aksjonspunkter}
-              sprakkode={sprakkode}
-              beregningResultat={resultatstruktur}
-              alleKodeverk={alleKodeverk}
-              tilbakekrevingvalg={tilbakekrevingvalg}
-            />
-          )}
+  const createPayload = harPotensieltFlereInformasjonsbehov(informasjonsbehovVedtaksbrev)
+    ? values => payloadMedEkstraInformasjon(values)
+    : values => payload(values);
 
-          {isAvslag(behandlingresultat.type.kode) && (
-            <VedtakAvslagPanel
-              behandlingStatusKode={behandlingStatusKode}
-              aksjonspunkter={aksjonspunkter}
-              behandlingsresultat={behandlingresultat}
+  const harRedusertUtbetaling = ytelseTypeKode === fagsakYtelseType.FRISINN;
+  const aktiverteInformasjonsbehov =
+    (informasjonsbehovVedtaksbrev?.informasjonsbehov || []).filter(({ type }) => type === 'FRITEKST') ?? [];
+  const mellomlagredeInformasjonsbehov = aktiverteInformasjonsbehov.map(informasjonsbehov => ({
+    [informasjonsbehov.kode]: dokumentdata?.[informasjonsbehov.kode] || '',
+  }));
+  return (
+    <>
+      <Formik
+        initialValues={Object.assign(
+          {},
+          {
+            [fieldnames.SKAL_BRUKE_OVERSTYRENDE_FRITEKST_BREV]:
+              kanKunVelge(tilgjengeligeVedtaksbrev, vedtaksbrevtype.AUTOMATISK) ||
```

Comment:
Dette betyr vel kanskje at boksen for overstyre fritekst vil være checked hvis det kun er AUTOMATISK brev?

Reply:
du har helt rett! gått litt fort i svingene her :) godt sett 👍